.github/workflows/downstream_tests.yml

```
@@ -393,3 +398,43 @@ jobs:
         run: |
           cd formulaic
           hatch run tests --config-file=pyproject.toml
+
+  validoopsie:
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.12"] # 3.9 and 3.12 are enough to cover all the tests
+        os: ["ubuntu-latest"]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
+        with:
+          enable-cache: "true"
+          cache-suffix: ${{ matrix.python-version }}
+          cache-dependency-glob: "pyproject.toml"
+      - name: clone-validoopsie
+        run: |
+          git clone https://github.com/akmalsoliev/Validoopsie.git --depth=1 validoopsie
+          cd validoopsie
```

Worth adding a `git log` command?

Mind adding a `git log`?
```suggestion
        run: |
          git clone https://github.com/akmalsoliev/Validoopsie.git --depth=1 validoopsie
          cd validoopsie
          git log
```